Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -0,0 +1,633 @@
+                    expect(stats.numberOfCommands).toEqual(2); // Renders two content tiles
+                });
+            });
+        });
+    });
+
+    it('loads tileset with external tiles.json', function() {
+        // Set view so that no tiles are loaded initially
+        viewNothing();
+
+        return Cesium3DTilesTester.loadTileset(scene, tilesetOfTilesetsUrl).then(function(tileset) {
+            // Root points to an external tiles.json and has no children until it is requested
+            var root = tileset._root;
+            expect(root.hasTilesetContent).toEqual(true);
+            expect(root.children.length).toEqual(0);
+
+            // Set view so that root's content is requested
+            viewRootOnly();
+            scene.renderForSpecs();
+            return root.readyPromise.then(function() {
+                // Root has one child now, the root of the external tileset
+                expect(root.children.length).toEqual(1);
+
+                // Check that headers are equal
+                var subtreeRoot = root.children[0];
+                expect(root.geometricError).toEqual(subtreeRoot.geometricError);
+                expect(root.refine).toEqual(subtreeRoot.refine);
+                expect(root._tileBoundingBox).toEqual(subtreeRoot._tileBoundingBox);
+
+                // Check that Subtree root has 4 children
+                expect(subtreeRoot.hasTilesetContent).toEqual(false);
+                expect(subtreeRoot.children.length).toEqual(4);
+            });
+        });
+    });
+
+    it('renders tileset with external tiles.json', function() {
+        return Cesium3DTilesTester.loadTileset(scene, tilesetOfTilesetsUrl).then(function(tileset) {
+            scene.renderForSpecs();
+            var stats = tileset._statistics;
+            expect(stats.visited).toEqual(7); // Visits two tiles with tileset content, five tiles with b3dm content
+            expect(stats.numberOfCommands).toEqual(5); // Render the five tiles with b3dm content
+        });
+    });
+
+    it('debugFreezeFrame', function() {
+        return Cesium3DTilesTester.loadTileset(scene, tilesetUrl).then(function(tileset) {
+            viewRootOnly();
+            scene.renderForSpecs();
+            var stats = tileset._statistics;
+            expect(stats.visited).toEqual(1);
+            expect(stats.numberOfCommands).toEqual(1);
+
+            tileset.debugFreezeFrame = true;
+            viewAllTiles();
+            scene.renderForSpecs();
+            expect(stats.visited).toEqual(0); // selectTiles returns early, so no tiles are visited
+            expect(stats.numberOfCommands).toEqual(1); // root tile is still in selectedTiles list
+        });
+    });
+
+    it('debugShowStatistics', function() {
+        return Cesium3DTilesTester.loadTileset(scene, tilesetUrl).then(function(tileset) {
+            tileset.debugShowStatistics = true;
+            scene.renderForSpecs();
+            // TODO : not sure how to test this
```

Comment:
Use a spy for console log.  Don't too tightly couple this test to the implementation though, just a coarse-grained test is plenty.